packages/app/src/components/css-reset.tsx

```diff
@@ -0,0 +1,344 @@
+import React from 'react';
+import { Global, css } from '@emotion/react';
+import { theme } from '@stacks/ui-theme';
+
+const defaultConfig = (theme: any) => ({
+  light: {
+    color: theme.colors.ink[900],
+    bg: undefined as any,
+    borderColor: '#E5E5EC', // TODO: replace this with theme color
+    placeholderColor: theme.colors.ink[400],
+  },
+  dark: {
+    color: 'white',
+    bg: theme.colors.ink[900],
+    borderColor: theme.colors.ink[600],
+    placeholderColor: theme.colors.ink[500],
+  },
+});
+const { color, bg, borderColor, placeholderColor } = defaultConfig(theme).light;
+
+const cssReset = css`
+  html {
+    line-height: 1.5;
+    color: ${color};
+    background-color: ${bg};
+    font-family: ${(theme as any)?.fonts?.body};
+  }
+
+  *,
+  *::before,
+  *::after {
+    box-sizing: border-box;
+    border-width: 0;
+    border-style: solid;
+    border-color: ${borderColor};
+  }
+
+  input:-ms-input-placeholder,
+  textarea:-ms-input-placeholder {
+    color: ${placeholderColor};
+  }
+
+  input::-ms-input-placeholder,
+  textarea::-ms-input-placeholder {
+    color: ${placeholderColor};
+  }
+
+  input::placeholder,
+  textarea::placeholder {
+    color: ${placeholderColor};
+  }
+`;
+
+const styles = css`
+  /*! modern-normalize v1.0.0 | MIT License | https://github.com/sindresorhus/modern-normalize */
+
+  /*
+Document
+========
+*/
+
+  /**
+Use a better box model (opinionated).
+*/
+
+  *,
```

Comment:
I don't think this is needed, there is already a declaration resetting `box-size` to `border-box` a few lines above.

Reply:
cc @aulneau - I'm sure you're right Andres!

Reply:
Yes there is a duplication here and above. You can probably remove the custom added bits above, and defer to the mondern-normalize for these two elements (box-sizing and line-height). However the line height above is different, so one of them is being overwritten